app/routes/wow.ilvl-export-search.tsx

```
@@ -0,0 +1,506 @@
+      if (itemNameFromId) {
+        setItemName(itemNameFromId)
+        setItemID(itemIdFromUrl)
+      }
+    } else {
+      setItemName('')
+      setItemID('')
+    }
+
+    setFormValues({
+      itemId: itemIdFromUrl || '',
+      ilvl: parseInt(ilvlFromUrl),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)